common/artifact_spawns/wc_artifact_spawns.txt

```diff
@@ -485,4 +485,253 @@ aluneth = {
     artifacts = {
 		aluneth = { value = 1 }
     }
+}
+felomelorn = {
+	max_amount = 1
+    spawn_date = -1.1.1
+	
+	spawn_chance = {
+		value = 100
+		modifier = {
+			factor = 0
+			NOT = { has_dlc = "Mystics" }
+		}
+	}
+    
+    weight = {
+		value = 0
+		additive_modifier = {
+			value = 100
+			character = 42002		# Anastarian
+		}
+	}
+    artifacts = {
+		felomelorn = { value = 1 }
+    }
+}
+hammer_of_the_twilight = {
+	max_amount = 1
+    spawn_date = 583.1.1
+	
+	spawn_chance = {
+		value = 100
+		modifier = {
+			factor = 0
+			NOT = { has_dlc = "Mystics" }
+		}
+		modifier = {
+			factor = 0
+			NOT = { year = 585 }
+		}
+	}
+    
+    weight = {
+		value = 0
+		additive_modifier = {
+			value = 100
+			is_playable = yes
+			
+			culture = twilights_hammer
+		}
+	}
+    artifacts = {
+		hammer_of_the_twilight = { value = 1 }
+    }
+}
+hammer_of_the_lightbringer = {
+	max_amount = 1
+    spawn_date = 583.1.1
+	
+	spawn_chance = {
+		value = 100
+		modifier = {
+			factor = 0
+			NOT = { has_dlc = "Mystics" }
+		}
+	}
+    
+    weight = {
+		value = 0
+		additive_modifier = {
+			value = 100
+			character = 60036						# Uther Lightbringer
+		}
+	}
+    artifacts = {
+		hammer_of_the_lightbringer = { value = 1 }
+    }
+}
```

Comment:
Has lightbringer hammer before becoming lightbringer

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/67305824-a8a88e00-f4fe-11e9-84be-4b99a4329d82.png)

</details>

Reply:
corrected. no spawn, but distributed by first-paladins-event